packages/table/src/Table.ts

```diff
@@ -447,7 +447,7 @@ export class Table extends SizedMixin(SpectrumElement, {
             this.tableBody = this.querySelector('sp-table-body') as TableBody;
             if (!this.tableBody) {
                 this.tableBody = document.createElement('sp-table-body');
-                this.tableBody.tabIndex = 0;
+                if (this.scroller) this.tableBody.tabIndex = 0;
```

Comment:
This is closer, but if you have a Table that is 1000px high and items that only add up to 500px there will be no scrollbar. We can confirm that with something like `element.offsetHeight < element.scrollHeight`. It's sadly something that likely needs an observer as it'll be something that could change over time... 🤔